### PR TITLE
User management

### PR DESCRIPTION
**Pull request by marek-vysoky**
Change User's Login, Create User, Delete Email Alias, Delete User, Get Current User, Groups - Get Memberships for User, Get User's Info, Invite User, Update User, Move User's Folder

### CODE REVIEW

**Comment by boxcla** [comment]
Verified that @marek-vysoky has signed the CLA. Thanks for the pull request!

**Comment by cburnette** [comment]
@coolboy Just a lot of stuff in this PR so need to review.

File: Box.V2/Models/BoxUser.cs [hunk 1]
```diff
@@ -24,7 +24,7 @@ public class BoxUser : BoxEntity
         public const string FieldJobTitle = "job_title";
         public const string FieldPhone = "phone";
         public const string FieldAddress = "address";
-        public const string FieldAvatarArl = "vatar_url";
+        public const string FieldAvatarUrl = "avatar_url";
```
Comment (coolboy) [inline Box.V2/Models/BoxUser.cs]: I think we should update it, but not sure the impact since this is public @cburnette

**Comment by coolboy** [comment]
@cburnette looks good to me with minor comment

**Comment by marek-vysoky** [comment]
@cburnette @coolboy Conflicts resolved. Can you check again?

**Pull request merged**